src/kernels/conv3x3.s

```diff
@@ -823,7 +824,7 @@ loop_begin:
   .load_input linesB, mbufs_cnt_B, linesB_start_offset
   .move_input_ptr 1
 
-  s_waitcnt vmcnt(1*mbufs_cnt_A) & lgkmcnt(0)
```

Comment:
This change doesn't look right. At this point we want to make sure that all linesA are ready and we are ok to have linesB still inflight. Isn't it?

Reply:
Sorry, I looked at the old version. The new one is correct.